db/schema.rb

```diff
@@ -713,6 +715,7 @@
     t.string  "role",        limit: 255, null: false
   end
 
+  add_index "user_roles", ["facility_id"], name: "fk_rails_dca27403dd", using: :btree
```

Comment:
I think I mentioned this before, but Oracle is kind of annoying and won't automatically generate indexes for foreign keys, so we need to manually add them as part of the migration.